### PR TITLE
Feat/inform when summary empty

### PR DESCRIPTION
**Pull request by joaoqueiroz95**
[Uploading disable-btn.webm…]()

### CODE REVIEW

**Comment by joaoqueiroz95** [comment]
[Booking Summary - Disable button if no tickets selected](https://trello.com/c/BPOK2JNM/82-booking-summary-disable-button-if-no-tickets-selected)

File: src/screens/__tests__/TicketSelection.test.tsx [hunk 1]
```diff
@@ -214,7 +214,23 @@ describe("Ticket Selection screen", () => {
     });
   });
 
-  it("should navigate to Summary screen when clicking on Summary button", async () => {
+  it("should not navigate to summary, when clicking on Summary button and no tickets is elected", async () => {
```
Comment (joel-gouveia) [inline src/screens/__tests__/TicketSelection.test.tsx:217]: why are tickets elected in this application? who are the voters?

**Review by joel-gouveia — CHANGES_REQUESTED** [review]
Minor text change, it can be merged afterwards

File: src/screens/__tests__/TicketSelection.test.tsx (hunk 1 above)
Comment (joaoqueiroz95) [reply src/screens/__tests__/TicketSelection.test.tsx:217]: I meant selected

**Review by mathvaillant — APPROVED** [review]

**Review by joel-gouveia — APPROVED** [review]

**Comment by mathvaillant** [comment]
LGTM!

**Comment by mathvaillant** [comment]
Tests fail. Missing mock of `useKeyboardEvents` at your `Cancellation.test.tsx`.

```
jest.mock("@hooks/useKeyboardEvents", () => {
  return {
    useKeyboardEvents: jest.fn(),
  };
});
```

**Pull request merged**